llama-index-core/llama_index/core/memory/chat_summary_memory_buffer.py

```diff
@@ -263,7 +263,18 @@ def _get_prompt_to_summarize(
         prompt = '"Transcript so far: '
         for msg in chat_history_to_be_summarized:
             prompt += msg.role + ": "
-            prompt += msg.content + "\n\n"
+            if msg.content:
+                prompt += msg.content + "\n\n"
+            else:
+                prompt += (
+                    "\n".join(
+                        [
+                            f"Calling function: {call.function.name} with args: {call.function.arguments}"

```

Comment:
I don't think we can guarantee that tool calls follow this structure. Anthropic, huggingface, openai, may all have different structures. 

Probably the safest is to just cast as a string?
`f"Calling a function: {str(call)}"`

Reply:
Thanks for advice! I'll update it.

Reply:
I've done that

Reply:
Thanks!